ownership/JAVA_DEPENDENCY_OWNERS.yaml

```diff
@@ -34,8 +34,8 @@ deps:
     artifact: bndlib
     owners:
 
-  ca.coglinc:javacc-gradle-plugin:
-    group: ca.coglinc
+  org.javacc.plugin:javacc-gradle-plugin:
+    group: org.javacc.plugin
     artifact: javacc-gradle-plugin
     owners:
```

Comment:
We just did away with owners files which is leading to this conflict, would you mind updating to pull in the changes from latest master?